test/screens/v2/candidate_generator/elevator/closures_test.exs

```diff
@@ -213,12 +214,6 @@ defmodule Screens.V2.CandidateGenerator.Elevator.ClosuresTest do
     end
 
     test "filters out alerts with no facilities or more than one facility", %{now: now} do
-      expect(@stop, :fetch_parent_station_name_map, fn ->
-        {:ok, %{"place-haecl" => "Haymarket"}}
-      end)
-
-      expect(@route, :fetch, fn _ -> {:ok, [%Route{id: "Red", type: :subway}]} end)
-
```

Comment:
Nice that pushing some of the fetch logic further down also simplified the setup of this test.